### ISSUE TITLE
Compatibility with ansible 2.4

### ISSUE THREAD

**Issue by alexppg** [metacloud/molecule#969]
# Issue Type

- Bug report

# Molecule and Ansible details

```
ansible --version
ansible 2.4.0.0
  config file = /home/asd/.ansible.cfg
  configured module search path = [u'/home/asd/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.13 (default, Jan 19 2017, 14:48:08) [GCC 6.3.0 20170118]
 
molecule --version
molecule, version 2.0.4
```

- Molecule installation method: pip
- Ansible installation method: pip

# Desired Behaviour

It should work with ansible 2.4.

# Actual Behaviour (Bug report only)
I've detected two things. The first one, the `--debug` flag doesn't work: 

```
--> Test matrix
    
├── mongodb_centos
│   ├── destroy
│   ├── dependency
│   ├── syntax
│   ├── create
│   ├── converge
│   ├── idempotence
│   ├── lint
│   ├── side_effect
│   ├── verify
│   └── destroy
├── mongodb_ubuntu
│   ├── destroy
│   ├── dependency
│   ├── syntax
│   ├── create
│   ├── converge
│   ├── idempotence
│   ├── lint
│   ├── side_effect
│   ├── verify
│   └── destroy
└── mongodb_debian
    ├── destroy
    ├── dependency
    ├── syntax
    ├── create
    ├── converge
    ├── idempotence
    ├── lint
    ├── side_effect
    ├── verify
    └── destroy
--> Scenario: 'mongodb_centos'
--> Action: 'destroy'
DEBUG: ANSIBLE ENVIRONMENT
---
ANSIBLE_CONFIG: /media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd/.molecule/ansible.cfg
ANSIBLE_FILTER_PLUGINS: /usr/local/lib/python2.7/dist-packages/molecule/provisioner/ansible/plugins/filters:/media/.syncthing/casa_trabajo/playbooks/roles/mongodb/plugins/filters:/media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd/.molecule/plugins/filters
ANSIBLE_LIBRARY: /usr/local/lib/python2.7/dist-packages/molecule/provisioner/ansible/plugins/libraries:/media/.syncthing/casa_trabajo/playbooks/roles/mongodb/library:/media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd/.molecule/library
ANSIBLE_ROLES_PATH: /media/.syncthing/casa_trabajo/playbooks/roles:/media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd/.molecule/roles

DEBUG: MOLECULE ENVIRONMENT
---
MOLECULE_DEBUG: 'True'
MOLECULE_DEPENDENCY_NAME: galaxy
MOLECULE_DRIVER_NAME: docker
MOLECULE_EPHEMERAL_DIRECTORY: /media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd/.molecule
MOLECULE_FILE: /media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd/molecule.yml
MOLECULE_INSTANCE_CONFIG: /media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd/.molecule/instance_config.yml
MOLECULE_INVENTORY_FILE: /media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd/.molecule/ansible_inventory.yml
MOLECULE_LINT_NAME: yamllint
MOLECULE_PROVISIONER_NAME: ansible
MOLECULE_SCENARIO_DIRECTORY: /media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd
MOLECULE_SCENARIO_NAME: mongodb_centos
MOLECULE_VERIFIER_NAME: testinfra

DEBUG: COMMAND
/usr/local/bin/ansible-playbook --vvv --inventory=/media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd/.molecule/ansible_inventory.yml --diff /media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd/destroy.yml -vv
Usage: ansible-playbook [options] playbook.yml [playbook2 ...]

ansible-playbook: error: no such option: --vvv
ERROR: 
```

In the other hand, when executing tests, it returns this error:

```
--> Executing Ansible Lint on /media/.syncthing/casa_trabajo/playbooks/roles/mongodb/molecule/centos_systemd/playbook.yml...
Traceback (most recent call last):
  File "/usr/local/bin/ansible-lint", line 7, in <module>
    from ansiblelint.main import main
  File "/usr/local/lib/python2.7/dist-packages/ansiblelint/__init__.py", line 28, in <module>
    import ansiblelint.utils
  File "/usr/local/lib/python2.7/dist-packages/ansiblelint/utils.py", line 53, in <module>
    from ansible.plugins import module_loader
ImportError: cannot import name module_loader
```

**Comment by alexppg** [comment]
I closed my eyes or something and didn't see that this issue is already opened. I'll close it.

**Comment by retr0h** [comment]
Molecule 2.4 released to pypi which supports Ansible 2.4.
